docs/src/main/sphinx/installation/kubernetes.rst

```diff
@@ -0,0 +1,342 @@
+
+Running Trino using Helm
+------------------------
+
+Run the following commands from the system with ``helm`` and ``kubectl``
+installed and configured to connect to your running Kubernetes cluster:
+
+#. Validate kubectl is pointing to the correct cluster by running the command:
+
+   .. code-block:: text
+
+      kubectl cluster-info --context=kind-trino
+
+   .. note::
+
+      The ``context`` parameter here is optional and references the cluster
+      created in the example using kind above. ``kubectl`` will use the
+      `default context <https://kubernetes.io/docs/reference/kubectl/cheatsheet/#kubectl-context-and-configuration>`_
+      if this parameter isn't supplied. Notice the context is the name of the
+      cluster with the ``kind-`` prefix added.
+
+   You should see similar output with a random port assigned if you are
+   running on kind.
+
+   .. code-block:: text
+
+      Kubernetes control plane is running at https://127.0.0.1:59553
+      CoreDNS is running at https://127.0.0.1:59553/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+
+#. (Optional) Create ``values.yaml`` file to define the parameters of your
+   deployment.
+
+   The helm chart has a `set of default values <https://github.com/trinodb/charts/blob/main/charts/trino/values.yaml>`_
+   that are used if you don't specify your own ``values.yaml`` file. When you
+   create this file, you will override only the values you specify. The
+   remaining properties will remain as the default. Add a ``values.yaml``
+   with the following configuration:
+
+   .. code-block:: yaml
+
+      image:
+        tag: "|trino_version|"
+      server:
+        workers: 2
+        config:
+          memory:
+            heapHeadroomPerNode: "1GB"
+          query:
+            maxMemory: "4GB"
+            maxMemoryPerNode: "1GB"
+      coordinator:
+        jvm:
+          maxHeapSize: "8G"
+      worker:
+        jvm:
+          maxHeapSize: "8G"
+
+   Let's explain what these values do and some useful notes.
+
+   #. ``.Values.image.tag`` is set to the current version, |trino_version|. Set
+      this value if you need to use a specific version of Trino. The default is
+      ``latest``.
+   #. ``.Values.server.workers`` is set to ``2``. This value sets the number of
+      workers, in this case, a coordinator and two worker nodes will get
+      deployed.
+   #. ``.Values.server.config.memory.heapHeadroomPerNode`` is set to ``1GB``.
+      This sets the :ref:`prop-resource-memory-heap-headroom-per-node` property.
+   #. ``.Values.server.config.query.maxMemory`` is set to ``4GB``.
+      This sets the :ref:`prop-resource-query-max-memory` property.
+   #. ``.Values.server.config.query.maxMemoryPerNode`` is set to ``1GB``.
+      This sets the :ref:`prop-resource-query-max-memory-per-node` property.
+   #. ``.Values.coordinator.jvm.maxHeapSize`` is set to ``8GB``.
+      This sets the ``Xmx`` property in the :ref:`jvm_config` on the
```

Comment:
`Xmx` is more cryptic than saying maximum heap memory that the cooridinator's JVM can use. Also most people interacting with Helm may not know what Xmx means. Either link to someplace to simplify.

Same for below.